table/meta_blocks.cc

```diff
@@ -59,12 +59,11 @@ PropertyBlockBuilder::PropertyBlockBuilder()
 
 void PropertyBlockBuilder::Add(const std::string& name,
                                const std::string& val) {
+  assert(props_.find(name) == props_.end());
```

Comment:
Why did we move this assertion from another Add() to this Add()?

Reply:
The other `Add` is an overload that will eventually call this `Add` too, it's moved here so it can catch both integer type properties and string type properties.

Reply:
ohhh I see - thanks!